### PR TITLE
[F] added optional s3 amazon caching (OFWEB-968)

### PR DESCRIPTION
**Pull request by SinisaG**
Added optional amazon s3 catching.

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "allowed_origins":  ["onefootball.com"],
+  "cache": {
+    "use": true,
+    "bucket":  "onefootball-photobooth"
```
Comment (maximevaly) [inline config/config.json]: Shouldn't we name that more specifically `aws_bucket`?

Comment (SinisaG) [inline config/config.json]: We can yeah. Or we can name it aws_cache. Then it's assumed that this is s3 bucket. :)

**Comment by SinisaG** [comment]
I will add also support for reading credentials from env variables, before merging.

**Comment by SinisaG** [comment]
> Remember, if you set your AWS credentials in the shared credentials file or via environment variables, the AWS SDK for Node.js will automatically detect them, and you will not need to perform any manual credential configuration in your application.

So it should be alright. I updated the docs to reflect that.

**Pull request merged**